packages/components/pagination/package.json

```diff
@@ -44,6 +44,15 @@
     "@nextui-org/react-utils": "workspace:*",
     "@nextui-org/shared-icons": "workspace:*",
     "@nextui-org/use-pagination": "workspace:*",
+    "@nextui-org/use-intersection-observer": "workspace:*",
+    "@nextui-org/tabs": "workspace:*",
+    "@nextui-org/table": "workspace:*",
+    "@nextui-org/input": "workspace:*",
+    "@nextui-org/button": "workspace:*",
+    "@nextui-org/dropdown": "workspace:*",
+    "@nextui-org/chip": "workspace:*",
+    "@nextui-org/user": "workspace:*",
```

Comment:
double check these dependencies. if they are for the stories / tests, put them to `devDependencies` instead.